ckanext/qa/helpers.py

```diff
@@ -3,6 +3,7 @@
 
 
 def qa_openness_stars_resource_html(resource):
+    print('-----> resource', resource)
```

Comment:
It's copied from the Keitaro fork. Initially I was hoping we could merge the work that they had done, but they ditched 2.8 support, so that didn't work out.